src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/properties.cpp

```diff
@@ -88,11 +88,23 @@ INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory, OVCheckChangePropComplieModleGetPr
                 ::testing::Values(ov::AnyMap({}))),
         OVCheckChangePropComplieModleGetPropTests_DEVICE_ID::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory, OVCheckChangePropComplieModleGetPropTests_InferencePrecision,
-        ::testing::Combine(
-                ::testing::Values(ov::test::utils::target_device),
-                ::testing::Values(ov::AnyMap({}))),
-        OVCheckChangePropComplieModleGetPropTests_InferencePrecision::getTestCaseName);
+/* Skip SW plugins (meta-plugins) in mandatory test suite */
+INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory,
+                         OVCheckChangePropComplieModleGetPropTests_InferencePrecision,
+                         ::testing::Combine(::testing::Values(sw_plugin_in_target_device(ov::test::utils::target_device) ?
```

Comment:
Please use the following construction instead of `skip` prefix:
```
                        ::testing::ValuesIn(OVCheckSetSupportedRWMetricsPropsTests::getRWMandatoryPropertiesValues(
                                {}, sw_plugin_in_target_device(ov::test::utils::target_device)))),
```